pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.black]
```

Comment:
It's ok the `pyproject.toml` but, in the **microservices-scaffold**, this configuration could be placed in `setup.cfg` and this file contains at this moment the configuration of the linters and other libaries, could be better put all the configuration in a single file :+1:

Reply:
Sorry, my fault, Black not support setup.cfg:

- https://github.com/psf/black/issues/683
- https://github.com/flying-sheep/awesome-python-packaging